keps/sig-node/2053-downward-api-hugepages/README.md

```diff
@@ -252,7 +252,8 @@ Not applicable
 
 ## Implementation History
 
-v1.20: Launch `Alpha` state
+- v1.27: Updated KEP to stable
```

Comment:
~~Did we skip over beta?~~
Ah, from PRR metadata it looks like 1.21 was beta but we missed it here.

Reply:
Added it to the history as suggested :+1: